concepts/ipc.md

```diff
@@ -76,7 +104,7 @@ There seem to be two main projects striving to provide dbus support for rust.
 * Wrapper library for libdbus -> libdbus dependency
 * Examples
 
-### Conclusion
+#### Conclusion
 Zbus seems to have some advantages over dbus-rs, mainly:
 * Official freedesktop.org library
 * Pure rust -> no ibdbus dependency
```

Comment:
Is the term "ibdbus" correct? I don't know it and i am not sure if it is a typo or not